packages/beacon-node/src/sync/backfill/backfill.ts

```diff
@@ -318,7 +318,7 @@ export class BackfillSync extends (EventEmitter as {new (): BackfillSyncEmitter}
       //       parent of
       //       the anchorBlock via strategy a) as it could be multiple skipped/orphaned slots
       //       behind
-      if (this.syncAnchor.lastBackSyncedBlock != null) {
+      if (this.syncAnchor.lastBackSyncedBlock !== null) {
```

Comment:
type of `this.syncAnchor.lastBackSyncedBlock` is `BackfillBlock | null`. Safe to use strict equality check